src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -374,11 +382,14 @@ public void run() {
                 dbq = q.take();
 
                 while (!dbq.stop) {
+                    //Too much logs!! let's not enable this for now...
+                    //logger.logWarning("Executing SqlBatch: " + dbq.queries, "SQLite");
```

Comment:
Do we need these comments here?

Reply:
This is just if we need to add even more logs. I kept this just to be easier for us if we need more logs.

Reply:
It's "dead code". If we need it we add it later. Keep the codebase clean.

Reply:
Done